src/util/error.js

```diff
@@ -23,8 +23,36 @@ export function getBadUserError(): Error {
 
 export function isUserCorrectableError(status: number) {
     return (
+        !status ||
         status === HTTP_STATUS_CODE_RATE_LIMIT ||
         status === HTTP_STATUS_CODE_UNAUTHORIZED ||
         status >= HTTP_STATUS_CODE_INTERNAL_SERVER_ERROR
     );
 }
+
+export function enhanceErrorForLogging(
+    error: Error,
+    type: ErrorTypes,
+    code: string,
+    contextInfo: Object = {},
+): ElementsError {
+    // $FlowFixMe
+    const enhancedError: ElementsError = error;
```

Comment:
I think reassigning here will still modify the original object. Did you mean to make a copy?

Reply:
It was intentional, I did not feel a deep clone was necessary. I think the error is pretty much thrown away after emitted

Reply:
In that case, you don't need the variable, right? Can you just add the property directly to `error`?

Reply:
It was mostly for flow typing